cloud-controller-manager/do/loadbalancers.go

```diff
@@ -1261,6 +1268,16 @@ func getEnableBackendKeepalive(service *v1.Service) (bool, error) {
 	return enableBackendKeepalive, nil
 }
 
+func getProjectID(service *v1.Service) string {
+	projectID, ok := service.Annotations[annDOLBProjectID]
+
+	if !ok {
+		return ""
+	}
+
+	return projectID
```

Comment:
nit: strictly speaking, since the fallback case is to just return an empty string anyway, this could be simplified to just `return service.Annotations[annDOLBProjectID]`